src/Simulation/Simulators.Tests/TestProjects/TargetedExe/TargetedExe.csproj

```diff
@@ -12,7 +12,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\..\CSharpGeneration\Microsoft.Quantum.CSharpGeneration.fsproj" PrivateAssets="All" IsQscReference="true" />
+    <PackageReference Include="Microsoft.Quantum.CSharpGeneration" Version="0.17.2106147878-beta" IsQscReference="true" />
```

Comment:
If we remove `<CSharpGeneration>false</CSharpGeneration>`, can we remove this `PackageReference`?